app/screens/home/recent_mentions/components/mention/mention.tsx

```diff
@@ -84,27 +97,46 @@ function Mention({post, currentUser}: Props) {
         />
     );
 
+    const handlePress = preventDoubleTap(() => {
+        pressDetected.current = true;
+
+        showPermalink(serverUrl, '', post.id, intl);
```

Comment:
why aren't you passing the team name ? in my mind it only needs to be an empty string if the post belongs to a DM/GM